src/routes/events.ts

```diff
@@ -95,9 +99,16 @@ router.put(
 // delete by id
 router.delete(
   '/:orgId/:id',
-  asyncHandler(async (req: Request, res: Response) => {
+  eventValidationRules,
```

Comment:
Delete shouldn't take any body. So what are these rules for?
```suggestion
```

Reply:
Ah probably accidentally added line 103-107.

Reply:
merge mistake :)